app/views/apps/_service_notification_fields.html.haml

```diff
@@ -15,12 +15,18 @@
       - NotificationService.subclasses.each do |notification_service|
         %div.notification_params{:class => (w.object.is_a?(notification_service) ? 'chosen ' : '') << notification_service.label}
           - notification_service::FIELDS.each do |field, field_info|
-            = w.label field, field_info[:label] || field.to_s.titleize
+            - field_type = field_info[:type] || :text_field
+            - value = field == :notify_at_notices ? w.object.notify_at_notices.join(", ") : w.object.send(field)
+
+            - if field_type == :check_box
+              %div.checkbox
+                = w.send field_type, field, :placeholder => field_info[:placeholder], :value => value
+                = w.label field, field_info[:label] || field.to_s.titleize
+            - else
+              = w.label field, field_info[:label] || field.to_s.titleize
+              = w.send field_type, field, :placeholder => field_info[:placeholder], :value => value
             - if field_info[:hint].present?
               %em= field_info[:hint]
-            - field_type = field == :password ? :password_field : :text_field
```

Comment:
This mapping of `:password` to `:password_field` went away, was this intentional?